in_toto/attestations.go

```diff
@@ -1,6 +1,7 @@
 package in_toto
 
 import (
+	ita1 "github.com/in-toto/attestation/go/v1"
```

Comment:
A `go mod tidy` is in order to add this to go.mod and go.sum.

Reply:
Yes! I need to run `fmt` and all that on this, too.